dice/platform_adapter_lagrange_helper.go

```diff
@@ -0,0 +1,281 @@
+package dice
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+	"runtime/debug"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+
+	"sealdice-core/utils/procs"
+)
+
+func lagrangeGetWorkDir(dice *Dice, conn *EndPointInfo) string {
+	workDir := filepath.Join(dice.BaseConfig.DataDir, conn.RelWorkDir)
+	return workDir
+}
+
+func NewLagrangeConnectInfoItem(account string) *EndPointInfo {
+	conn := new(EndPointInfo)
+	conn.ID = uuid.New().String()
+	conn.Platform = "QQ"
+	conn.ProtocolType = "onebot"
+	conn.Enable = false
+	conn.RelWorkDir = "extra/lagrange-qq" + account
+
+	conn.Adapter = &PlatformAdapterGocq{
+		EndPoint:          conn,
+		UseInPackGoCqhttp: true,
+		BuiltinMode:       "lagrange",
+	}
+	return conn
+}
+func LagrangeServe(dice *Dice, conn *EndPointInfo, loginInfo GoCqhttpLoginInfo) {
+	pa := conn.Adapter.(*PlatformAdapterGocq)
+
+	pa.CurLoginIndex++
+	loginIndex := pa.CurLoginIndex
+	pa.GoCqhttpState = StateCodeInLogin
+
+	if pa.UseInPackGoCqhttp && pa.BuiltinMode == "lagrange" { //nolint:nestif
+		workDir := lagrangeGetWorkDir(dice, conn)
+		_ = os.MkdirAll(workDir, 0o755)
+
+		exeFilePath, _ := filepath.Abs(filepath.Join(workDir, "Lagrange.OneBot"))
+		exeFilePath = strings.ReplaceAll(exeFilePath, "\\", "/") // windows平台需要这个替换
```

Comment:
考虑用 `filepath.ToSlash`？


```suggestion
		exeFilePath = filepath.ToSlash(exeFilePath) // windows平台需要这个替换
```